### DISCUSSION TITLE
Bi-Weekly Community Meeting, Saturday, June 15, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3477] [Announcements]
Everyone is invited to the next TaskWarrior community bi-weekly meeting!  
Want to get a sense of what to expect? Watch the recent meeting videos here…

- [May 4](https://youtu.be/gcZ6MZYA_88)
- [May 18](https://www.youtube.com/watch?v=r5G9yiDrgkg)
- [June 1](https://youtu.be/hZQG8jDn24I)

Our next meeting is 12:00 UTC Saturday, June 15, 2024 ([check that in your time zone @ timeanddate.com](https://www.timeanddate.com/worldclock/meetingtime.html?month=6&day=15&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0)). You can [add this event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MzdqaDhwNDc4dHE0bGd0b3M3aWs5anFwbG8gd2VzbGV5QG13cGVubmVyLmNvbQ&tmsrc=wesley%40mwpenner.com).

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting runs ~30 minutes. The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.

**Comment by wrashi** [comment]
Just wanted to renew the invitation to join the Community Meeting call tomorrow. Everyone is welcome!

**Comment by wrashi** [comment]
The meeting room is now open for anyone who wants to join a bit early…

**Reply by wrashi** [reply]
Thanks to everyone who showed up for yesterday's excellent meet up! The video is available here:
https://youtu.be/qI9P1L8fg0s

Matt described how he syncs next tasks to his phone
Dustin, Felix & Thomas provided updates on recent work and their focus for the next weeks
Thomas & Matt showed off website changes and discussed further changes

**Reply by djmitche** [reply]
Thanks for organizing! These meetings are, I think, helping to build community in the project and establish momentum.
